backend/src/main/java/com/woowacourse/momo/member/service/MemberValidator.java

```diff
@@ -0,0 +1,28 @@
+package com.woowacourse.momo.member.service;
+
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.RequiredArgsConstructor;
+
+import com.woowacourse.momo.member.domain.Member;
+import com.woowacourse.momo.member.domain.MemberRepository;
+import com.woowacourse.momo.member.exception.MemberErrorCode;
+import com.woowacourse.momo.member.exception.MemberException;
+
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+@Service
+public class MemberValidator {
+
+    private final MemberRepository memberRepository;
+
+    public void validateExistMember(Long memberId) {
+        Optional<Member> member = memberRepository.findById(memberId);
+        if (member.isEmpty() || member.get().isDeleted()) {
+            throw new MemberException(MemberErrorCode.MEMBER_NOT_EXIST);
+        }
```

Comment:
```suggestion
        Member member = memberRepository.findById(memberId)
                .orElseThrow(() -> new MemberException(MemberErrorCode.MEMBER_NOT_EXIST));

        if (member.isDeleted()) {
            throw new MemberException(MemberErrorCode.MEMBER_NOT_EXIST);
        }
```
`회원이 존재하지 않음`과 `삭제되었음`은 관심사가 다르기 때문에 예외 throw를 달리두는 것이 좋아보여요!!

Reply:
해당 피드백도 반영하였습니다!